consts.js

```diff
@@ -0,0 +1,7 @@
+const CREATE_REPO_FAILED_MESSAGE = "Github only returned error \"Not Found\". Perhaps your access token provides insufficient access? The required scope to create a new repository is \"repo\" (full control of private repositories).";
+const CREATE_ORG_WEBHOOK_FAILED_MESSAGE = "Github only returned error \"Not Found\". Perhaps your access token provides insufficient access? The required scope to create an organization webhook is \"admin:org_hook\" (full control of organization hooks).";
```

Comment:
Both these errors' names indicate that they report any error of the particular operation, while the message explicitly refers to `Not Found`.

These are `...ERROR_MESSAGES` not some generic `..._MESSAGE`s

`Github returned \"Not Found\" error only` sounds better, or if it's too posh then `Github returned only \"Not Found\" error`

`Perhaps your access token provides insufficient access?` - the sentence expresses uncertainty twice, once with `perhaps`, second time with the question form. Using just one is enough.